python/README.md

```diff
@@ -0,0 +1,15 @@
+# Python Examples
+Python Pub/Sub Examples for HPE Ezmeral Data Fabric Topics
+
+## Install kafka-python: pip3 install kafka-python
+`pip3 install kafka-python`
+`pip3 list | grep kafka-python`
+
+### Running Examples
```

Comment:
Does the user need to create a stream ahead of time? How will they do this? What permissions do they need to set?

IT looks from the commit message like you committed this program while running as root. That might have obscured permissions issues.